cli/cmd/inject.go

```diff
@@ -566,20 +571,20 @@ func injectResource(bytes []byte, options *injectOptions, report *injectReport)
 		// Lists are a little different than the other types. There's no immediate
 		// pod template. Because of this, we do a recursive call for each element
 		// in the list (instead of just marshaling the injected pod template).
-
-		// TODO: generate an injectReport per list item
-		return injectList(bytes, options, report)
-
+		return injectList(bytes, options)
 	}
 
+	report := injectReport{}
+	report.name = fmt.Sprintf("%s/%s", strings.ToLower(meta.Kind), om.Name)
```

Comment:
TIOLI -- you could set the name field here on initialization, rather than as a separate operation:

```
	report := injectReport{
		name: fmt.Sprintf("%s/%s", strings.ToLower(meta.Kind), om.Name),
	}
```

Reply:
Good catch :+1: